src/jdk.compiler/share/classes/module-info.java

```diff
@@ -162,6 +162,7 @@
  * <tr><th scope="row">{@code fallthrough}          <td>falling through from one case of a {@code switch} statement to
  *                                                      the next
  * <tr><th scope="row">{@code finally}              <td>{@code finally} clauses that do not terminate normally
+ * <tr><th scope="row">{@code lossy-conversions}    <td>possible lossy conversions in compound assignment
```

Comment:
In the table where there is more room to accommodate text, perhaps a longer description mentioning primitive types could be used.